routes/sendMoney.js

```diff
@@ -0,0 +1,76 @@
+var express = require("express");
+var router = express();
+var database = require("../database");
+
+const bodyParser = require("body-parser");
+
+router.use(express.json());
+router.use(bodyParser.json());
+router.use(
+  bodyParser.urlencoded({
+    extended: true,
+  })
+);
+
+router.post("/sendmoney", async function (request, response) {
+  const data = await request.body;
+
+  if (data.sender_id && data.receiver_id && data.amount) {
+    database.query(
+      "select balance from wallet where user_id = ?",
+      [data.sender_id],
+      function (err, results) {
```

Comment:
move this try to cover whole code, so if any error is encountered it will be taken care of in catch.